ocpp/routing.py

```diff
@@ -90,25 +96,26 @@ def after_boot_notification(self, *args, **kwargs):
 
     """
     routes = {}
-    for attr_name in dir(obj):
-        attr = getattr(obj, attr_name)
-        for option in ['_on_action', '_after_action']:
-            try:
-                action = getattr(attr, option)
-
-                if action not in routes:
-                    routes[action] = {}
-
-                # Routes decorated with the `@on()` decorator can be configured
-                # to skip validation of the input and output. For more info see
-                # the docstring of `on()`.
-                if option == '_on_action':
-                    routes[action]['_skip_schema_validation'] = \
-                        getattr(attr, '_skip_schema_validation', False)
-
-                routes[action][option] = attr
-
-            except AttributeError:
-                continue
+    for attr_name in routables:
+        if hasattr(obj, attr_name):
```

Comment:
I think we dont need this check, since the decorator on and after is always called in methods from the same class where the create_route_map is called from. 

If we want to keep a check, then I guess we can move the try/except for AttributeError to include the code
```python
attr = getattr(obj, attr_name)
```

Reply:
it was needed, otherwise the tests for the charge points (test_v16_charge_point and v20) would fail. I think its due to the list not being emptied between the test cases or similar, did not look into it super deep. But, it is a good ide to move the code to the try clause, will fix that